cmds/completions/completions.go

```diff
@@ -0,0 +1,31 @@
+package completions
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/taskcluster/taskcluster-cli/cmds/root"
+)
+
+var (
+	defaultFilename = "bash_completion.sh"
+)
+
+func init() {
+	// Add the task subtree to the root.
+	use := "completions <filename (default:" + defaultFilename + ")>"
+	completionsCommand := &cobra.Command{
+		Short: "Provides bash completion script.",
+		RunE:  genCompletion,
+		Use:   use,
+	}
```

Comment:
So could I do:
`$ sudo taskcluster completions /etc/bash_completion.d/taskcluster.sh`

to install these completions globally?

Maybe we could put that in the `--help` message, or even make the default filename on linux... :)

Reply:
I didn't necessarily want to make the default output file one you needed special rights to write to, but adding the instructions to the help sounds like a good idea. Macs are... special

Reply:
oh, yeah, good point..